main.py

```diff
@@ -1,7 +1,37 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genre_names = ["Western", "Action", "Dramma"]
+    for genre_name in genre_names:
```

Comment:
always add blank lines before and after for-loops, and if-statements. Thus your code looks more readable

Reply:
fixed